Program.cs

```diff
@@ -3,17 +3,204 @@
+         SetCursorPosition (mCurrentColumn, mCurrentRow);
+         for (int i = 0; i < 5; i++) { // Setting the corresponding colors for letters.
+            char letter = mLetters[i];
+            if (letter == mWordleWord[i])
+               mKeyboard[letter] = Green;
+            else if (mWordleWord.Contains (letter) && mKeyboard[letter] != Green)
+               mKeyboard[letter] = Yellow;
+            else if (mLetters.Contains (letter) && !mWordleWord.Contains (letter))
+               mKeyboard[letter] = Red;
+         }
+
+         for (char letter = 'A'; letter <= 'Z'; letter++) { // Having 9 alphabets in a row.
+            if (letter is 'J' or 'S') {
+               WriteLine ();
+               SetCursorPosition (sKeyBoardColumn, mCurrentRow += 2); // Having keyboard row gap as 2 lines.
+            }
+            ChangeConsoleColor (mKeyboard[letter]);
+            Write (letter + sSpaces);
+         }
+         ResetColor ();
+         WriteLine ();
+      }
+
+      /// <summary>Prints the message in corresponding color</summary>
+      /// <param name="color">Color of the message</param>
+      /// <param name="message">The message to be printed</param>
+      static void PrintMessage (EColor color, string message) {
+         SetCursorPosition (sKeyBoardColumn, sRow * 10);
+         ChangeConsoleColor (color);
+         Write (color == Yellow ? $"{message,25}" : $"{message,40}");
+         ResetColor ();
+      }
+
+      /// <summary>Changes the ForegroundColor as required</summary>
+      /// <param name="color">Color to which the ForegroundColor must be changed</param>
+      static void ChangeConsoleColor (EColor color) {
+         ForegroundColor = color switch {
+            Green => ConsoleColor.Green,
+            DarkGray => ConsoleColor.DarkGray,
+            Red => ConsoleColor.Red,
+            Yellow => ConsoleColor.Yellow,
+            White => ConsoleColor.White,
+            _ => ConsoleColor.Black,
+         };
+      }
+      #endregion
+
+      #region Enum --------------------------------------------------
+      public enum EColor { DarkGray, Red, Green, Yellow, White, Black }
+      #endregion
+
+      #region Members -----------------------------------------------
+      static char[] mLetters = new char[5]; // Entry characters
+      static int mKeyColumn, mKeyRow; // Position at which a key is pressed.
+      static string mWordleWord, mUserWord; // Random word to be picked by computer, user typed word respectively.
+      static Dictionary<char, EColor> mKeyboard = new (); // Keyboard letters and their corresponding colors
+      #endregion
+
+      #region Private -----------------------------------------------
+      static char sDot = '\u00b7', sCircle = '\u25cc';
+      static string sSpaces = "    "; // 4 spaces
+      static string[] sWords = File.ReadAllLines ("C:\\WorkGIT\\Training\\puzzle.txt"); // Wordle words

```

Comment:
Though you have embedded dict and puzzle files, you are referring here the location in your system. So it crashes.